includes/Admin/Abstract_Settings_Screen.php

```diff
@@ -104,7 +104,9 @@ protected function is_current_screen_page() {
 			return false;
 		}
 		// assume we are on the Connection tab by default because the link under Marketing doesn't include the tab query arg
-		$tab = Helper::get_requested_value( 'tab', 'connection' );
+		$connection_handler = facebook_for_woocommerce()->get_connection_handler();
+		$default_tab = $connection_handler->is_connected() ? 'advertise' : 'connection';
+		$tab = Helper::get_requested_value( 'tab', $default_tab );
```

Comment:
```suggestion
		$connection_handler = facebook_for_woocommerce()->get_connection_handler();
		$default_tab        = $connection_handler->is_connected() ? 'advertise' : 'connection';
		$tab                = Helper::get_requested_value( 'tab', $default_tab );
```

Small coding style fixes